File: app/src/main/java/com/mobile/model/PopInfo.java
package com.mobile.model;

/**
 * Created by anubis on 7/17/17.
 */

public class PopInfo {

    int reservationId;
    String movieTitle;
    String theaterName;
    String tribuneTheaterId;
    String showtime;
    String tribuneMovieId;

    public PopInfo() {
    }

    public String getMovieTitle() { return movieTitle; }

    public String getTribuneMovieId() { return tribuneMovieId; }

    public int getReservationId() { return reservationId; }

    public String getTheaterName() { return theaterName; }

    public String getTribuneTheaterId() { return tribuneTheaterId; }

    public String getShowtime() { return showtime; }
}

File: app/src/main/java/com/mobile/model/PopInfo.kt
package com.mobile.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by anubis on 7/17/17.
 */
@Parcelize
data class PopInfo(
        var reservationId: Int = 0,
        var movieTitle: String? = null,
        var theaterName: String? = null,
        var tribuneTheaterId: String? = null,
        var showtime: String? = null,
        var tribuneMovieId: String? = null
) : Parcelable